pom.xml

```diff
@@ -52,6 +52,8 @@
         <slf4jVersion>1.7.25</slf4jVersion>
         <springVersion>5.0.16.RELEASE</springVersion>
         <springBootVersion>2.0.5.RELEASE</springBootVersion>
+        <springVersion>5.3.19</springVersion>
+        <springBootVersion>2.6.7</springBootVersion>
```

Comment:
I presume you meant to update the two existing properties rather than adding these as two new ones.